components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Extensions/PublishedDataItemsModelEx.cs

```diff
@@ -31,9 +31,9 @@ public static class PublishedDataItemsModelEx {
                     continue;
                 }
                 var monitoredItem = item.ToMonitoredItem();
-                map.Add(monitoredItem.Id ?? Guid.NewGuid().ToString(),
-                    monitoredItem);
-                if (item.HeartbeatInterval == null) {
+                map.Add(monitoredItem.Id ?? Guid.NewGuid().ToString(), monitoredItem);
+                /*  Heartbeat using triggering mode
```

Comment:
If we never believe this will be using triggering lets just remove the commented out code.

Reply:
let's keep it as sample for generic triggering feature, I think it makes a lot of sense to promote this feature.